<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunnelingFeatures</class>
 <widget class="QWidget" name="TunnelingFeatures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Tunnelling Service Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="tunnelServiceType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>16</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>TunnelingFeatureGet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TunnelingFeatureSet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Feature Identifier:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="featureIdentifier">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Supported EMI type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Host Device Device Descriptor Type 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bus connection status</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KNX Manufacturer Code</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Active EMI type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interface Individual Address</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maximum APDU Length</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interface Feature Info service Enable</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Feature value (HEX):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="featureValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="tunnelingSend">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textOuputTunneling">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="secureSessionCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use secure session</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="secureSessionCb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectTunneling">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectTunneling">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tunnelServiceType</tabstop>
  <tabstop>featureIdentifier</tabstop>
  <tabstop>featureValue</tabstop>
  <tabstop>tunnelingSend</tabstop>
  <tabstop>textOuputTunneling</tabstop>
  <tabstop>secureSessionCheckBox</tabstop>
  <tabstop>secureSessionCb</tabstop>
  <tabstop>connectTunneling</tabstop>
  <tabstop>disconnectTunneling</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>secureSessionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>secureSessionCb</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
